### PR TITLE
Remove tile_images and special_materials obsolete code

### PR DESCRIPTION
**Pull request by Zughy**
Git blaming has brought me here https://github.com/minetest/minetest/commit/fd1135c7af46eb2f5b99a11f48bf9f9ae335ea9c
## To do

This PR is Ready for Review.

## How to test
Go back in time and be sure it was Celeron the one who wrote those lines

### CODE REVIEW

**Comment by SmallJoker** [comment]
There's also code on C++ side. What's going to happen with that? Exist forever?

**Comment by Zughy** [comment]
Silly me, removed it

File: builtin/game/item.lua [hunk 1]
```diff
@@ -597,13 +597,11 @@ core.nodedef_default = {
...
-	--tiles ={""},
-	--special_tiles = {
-	--	{name="", backface_culling=true},
-	--	{name="", backface_culling=true},
-	--},
+	tiles ={""},
+	special_tiles = {
+		{name="", backface_culling=true},
+		{name="", backface_culling=true},
+	},
```
Comment (sfan5) [inline builtin/game/item.lua:604]: Don't think there's much value in having these stubs (also there can be more than two special tiles, this code is probably old).
Suggestion:
```lua
	tiles = {},
	special_tiles = {},
```

**Review by SmallJoker — APPROVED** [review]
Works.

**Comment by appgurueu** [comment]
Yay! The code base is now 0.01% cleaner /s.

**Review by sfan5 — APPROVED** [review]

**Pull request merged**